.pylintrc

```diff
@@ -82,3 +90,14 @@ disable=
 	# if:/else: and for:/else:.
 	useless-else-on-loop,
 	no-else-return,
+	# NOTE(lidiz): Python 3 make object inheritance default, but not PY2
+	useless-object-inheritance,
+	# NOTE(lidiz): Limit the length of function doesn't mean reducing
+	# complexity.
+	too-many-statements,
+	# NOTE(https://github.com/PyCQA/pylint/issues/1498)
+	unsubscriptable-object,
+	# TODO(https://github.com/grpc/grpc/issues/17380)
```

Comment:
Please point to this TODO on a per-line basis. I only see `non-iterator-returned` and `stop-iteration-return` in a very small number of (mostly test) files.